_data/simple-icons.json

```
@@ -1551,6 +1551,11 @@
             "hex": "334455",
             "source": "https://www.autohotkey.com"
         },
+        {
+            "title": "AutoIt Script",
```

It looks like the brand name is actually just "AutoIt".
```suggestion
            "title": "AutoIt",
```